README.md

```diff
@@ -183,3 +182,85 @@ response:
   location: the/new/location
 ---
 ```
+
+## Sitemap application
+
+### Build
+
+The script ``map.py`` builds a conglomerate site-map from a set of marXdown
+sites. It is assumed that these sites are under Git version control, and that
+the person/process running this script has authorization to read from them.
+
+This script reads from a config file that contains a set of site
+specifications. Here is an example:
+
+```json
+   {
+     "sites": [
+       {
+         "name": "help",
+         "repo": "git@github.com:arXiv/arxiv-docs.git",
+         "source_dir": "help",
+         "source_ref": "develop",
+         "human_name": "arXiv Help",
+         "url_prefix": "/help"
+       },
+       {
+         "name": "labs",
+         "repo": "git@github.com:arXiv/arxiv-docs.git",
+         "source_dir": "labs",
+         "source_ref": "develop",
+         "human_name": "arXiv Labs",
+         "url_prefix": "",
+         "server": "https://labs.arxiv.org"
+       }
+     ]
+   }
+```
+
+
+Most of these keys correspond to variables used in the marXdown application:
+
+``name``
+    Sets the ``SITE_NAME`` used when building the marXdown site.
+``repo``
+    Full path to the Git repository containing the marXdown site.
+``source_dir``
+    Relative path within the Git repository, used to build the full
+    ``SOURCE_PATH`` once the repo is cloned locally.
+``human_name``
+    Sets the ``SITE_HUMAN_NAME`` used when building the marXdown site, and
+    is used as the title of the root of the site's tree if there is no
+    root index page.
+``url_prefix``
+    Sets the ``SITE_URL_PREFIX`` used when building the marXdown site.
+``server``
+    Optional. The base URL (not including the ``url_prefix``) where the site
+    is deployed in production. If provided, paths in the site's tree will be
+    prefixed with this server address.
+
+
+To build the sitemap:
+
+```bash
+pipenv run python map.py -s /path/to/sites.json -o /path/to/map.json
```

Comment:
It tripped me up that I needed to omit the leading slash on paths - is it against convention to use `path/to/sites.json` rather than `/path/to/sites.json`?

Reply:
Only need to omit if it's a relative path, e.g. ``samples/sites.json`` would be the relative path; on my machine the absolute path is ``/Users/brp53/projects/arxiv-markdown/samples/sites.json``. Is that what you mean?